sdk/servicebus/service-bus/samples/typescript/gettingStarted/scheduledMessages.ts

```diff
@@ -72,13 +72,12 @@ async function receiveMessages(sbClient: ServiceBusClient): Promise<void> {
   const queueClient = sbClient.createQueueClient(queueName);
 
   let numOfMessagesReceived = 0;
-  const onMessageHandler: OnMessage = async (brokeredMessage) => {
+  const onMessageHandler: OnMessage = async (brokeredMessage): Promise<void> => {
```

Comment:
We added `allow-expressions` to allow untyped async function expressions didn't we?

Reply:
As long as the destination variable is typed (which it is, `OnMessage`) you shouldn't need to annotate the return type of the async function here.

Reply:
we did add `allow-expressions` but it was still giving the explicit function return type error

Reply:
This should be allowed as long as the `allowTypedFunctionExpressions` option is on for that rule.

Reply:
I added `allowtypedfunctionexpressions` and fixed a few of the statements with the return, 
And resolved some with the type annotations. One of them didn't resolve with the type annotations, so added the return type specifically. For a few I don't know the correct types, so will ask @ramya-rao-a  about them in the comments